web/templates/play/show.html.eex

```diff
@@ -1,2 +1,4 @@
 <%= render "_modal_keybinds.html" %>
 <div id="board-viewer" class="play-show"></div>
+
+<%= link "Explodey View", to: skin_path(@conn, :show, @game), class: "btn btn-default btn-xs view-btn" %>
```

Comment:
Links from the a board rendering view are unnecessary and only clutter the page,  let's keep them all on the resource index or show pages.